docs/pages/Hosting/LocalTesting.adoc

```diff
@@ -0,0 +1,47 @@
+= Testing netlify deployments locally
+
+. <<Introduction>>
+. <<The Netlify CLI>>
+.. <<Installation>>
+.. <<Further Reading>>
+
+== Introduction
+While you can test your Netlify deployments by pushing to a branch and then deploying that branch to a preview URL, it's often useful to test your builds locally before pushing to a remote branch. 
+This is especially true if you're working on a feature branch that you don't want to push to a remote branch until it's ready for review.
+You can run your PWA locally using the nuxt dev or npm run dev commands, but this won't run in the same environment as when deployed on netlify.
```

Comment:
Testing locally, doesn't provide the exact same environment as deploying it on netlify, since it's not serverless when running locally. => I'd suggest to rewrite this sentence